### DISCUSSION TITLE
Poll for new name

### DISCUSSION THREAD

**Discussion by jgadsden** [maxGraph/maxGraph#43] [Development]
@coclav , @mcyph , @lreading , @junsikshim , @tbouffard , @Kurt29 , @csouchet would you all like to vote for a name for this project? Contacting you because you have all responded to #23 .

As suggested by Colin, vote for the one you want with a thumbs up or whatever. I guess we keep the voting open for a week or so, or until everybody has voted

Note that Céline points out that  tsGraph already exists so it has to be ruled out, and also for jsGraph

**Comment by jgadsden** [comment]
maxGraph

**Comment by jgadsden** [comment]
freeGraph

**Comment by jgadsden** [comment]
veGraph

**Comment by jgadsden** [comment]
draw-graph

**Comment by jgadsden** [comment]
Disqualified : jsGraph

**Reply by jgadsden** [reply]
Sorry  @lreading , we can not consider jsGraph as it is already in use (see comment below)

**Reply by lreading** [reply]
I don't have a super strong preference/opinion for the new name. :) Just wanted to participate to keep the momentum going!

**Comment by cueman** [comment]
jsGraph already exists as well (first Google hit is node-jsgraph and the second jsgraph - both on npmjs.com)

I'm new to this project having only very recently looked for this kind of thing. So maybe my question isn't pertinent...but where does the suggestion "veGraph" come from? And why "maxGraph" - is this because it's close to the old name or something deeper?

**Reply by jgadsden** [reply]
Hello @cueman - the discussion about names is in #23 . Many thanks for pointing out that jsgraph exists, I guess that removes it from the running: https://www.npmjs.com/package/jsgraph

veGraph came from https://github.com/jsGraph/mxgraph/discussions/23#discussioncomment-620923

and nothing particularly deep about maxGraph, just memorable https://github.com/jsGraph/mxgraph/discussions/23#discussioncomment-620467

**Reply by jgadsden** [reply]
@cueman did you want to vote for a name? Please do :-)

**Comment by jgadsden** [comment]
Well, it looks like maxGraph is the run away favourite.
@coclav do you want to rename the repo? I can do it if you like?

**Reply by coclav** [reply]
let's go ! 

shall we also rename the github "organisation" ? or we keep it jsGraph/maxGraph ?

**Reply by jgadsden** [reply]
Hello @coclav - as you were the driver for this and it is your organisation then I think you have the honour of deciding what name for the organisation :-)

Personally 'jsGraph' works for me OK, but I will go with whatever you think.

**Comment by jgadsden** [comment]
@coclav just checking what you would like to do:

I suggest rename 'mxgraph' -> 'maxgraph' but are you thinking camel case? ie 'mxgraph' -> 'maxGraph' ?

**Comment by coclav** [comment]
All done !! 

I have renamed the organisation (max-graph) and the project (maxGraph) so we can start off fresh :) 

I opted for styled "maxGraph" as it's more readable. I find. Unfortunately the organisation was not available but i still wanted the consistency, so we'll have to live with a little "-" in our name

onwards! 

Thanks everyone for your contributions. cc @jgadsden

**Reply by tbouffard** [reply]
Very good news, thanks!

About the organization name, it seems it has been created on November by @coclav-skore probably linked to you @coclav 😺 See https://github.com/maxGraph/maxgraph/issues/1
If I am right, I guess you can do something to drop the existing `maxGraph` organization and then rename our `max-graph` organization into `maxGraph`.

**Reply by coclav** [reply]
Oh dear how did i end up in a situation like that ?!

anyway all renamed clean and we don't change it anymore :-)
